python/stlib/physics/deformable/elasticmaterialobject.py

```diff
@@ -43,6 +43,8 @@ def createPrefab(self,
                      youngModulus=18000,
                      totalMass=1.0, solver=None):
 
+        self.createObject("RequiredPlugin", name="SofaSparseSolver")
```

Comment:
I'm not sure we should add the requiredplugin in the object.

Reply:
I also find it strange to put that in the object, but where else?

Reply:
I'm glad you ask.
Currently this should go in the Scene(plugins=["SofaSparseSolver"]) prefab. The good thing by doing such a thing is that when loading a scene the dependencies are clear. 

The drawback is that the scene dependencies are handled 'per scene' instead of 'per component'.

To defer to each prefab a way to indicate their dependencies. We could
```python
def ElasticMaterialObject(parent):
          if not parent.getRoot().plugins.hasObject("SofaSparseSolver"):
              print("Missing SofaSparseSolver in the scene.") 
              parent.getRoot().plugins.createObject("SofaSparseSolver")
```

There is probably other way to do that and I dont know what is the best. 
We can discuss that during the coding sprint.